src/hotspot/os_cpu/bsd_zero/javaThread_bsd_zero.cpp

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 1997, 2025, Oracle and/or its affiliates. All rights reserved.
- * Copyright 2009, 2010 Red Hat, Inc.
+ * Copyright (c) 2009, 2021, Red Hat, Inc. All rights reserved.
```

Comment:
I don't think you should be touching Red Hat copyright lines like this..?

Reply:
No, that's clearly wrong. Probably a result of comparing between master and jdk23u or jdk24u.